PersonStageThree/src/com/beyond/person/core/exception/InvalidAgeException.java

```diff
@@ -0,0 +1,7 @@
+package com.beyond.person.core.exception;
```

Comment:
@maria-salmasian where is readme ? where is diagram ? comments need to be improved , there is no need to write comment on getter . Also you should write comment on logic where it will be useful

Reply:
there is a readme with flowchart and uml on this branch (stage3)